providers/hns/hns_roce_u.h

```diff
@@ -119,7 +119,7 @@ struct hns_roce_buf {
 };
 
 #define BIT_CNT_PER_BYTE       8
-#define BIT_CNT_PER_U64		64
+#define BIT_CNT_PER_LONG       (BIT_CNT_PER_BYTE * sizeof(unsigned long))
```

Comment:
The change doesn't align with commit message.
Please explain why are you doing what are you doing.
Why did you change variable types from clean uint32_t/uin64_t to long?
Why did you mix them?

Reply:
This macro is for bitmap in srq, and it's defined in type of unsigned long, so I think PER_LONG is better than PER_U64. Is it ok?
Sorry for that I didn't mentioned it in the commit message, will modify it and pay more attention on the commit message.